assign1_1.js

```diff
@@ -0,0 +1,13 @@
+// Problem 1: Complete the secondLargest function which takes in an array of numbers in input and return the second biggest number in the array. (without using sort)?
+function secondLargest(array) {
```

Comment:
Space after function name 
Why cant you address all the linting error in same PR?